_includes/selector.html

```diff
@@ -452,18 +440,22 @@
                 var pkgs = this.active_packages;
                 var py_cuda_pkgs = [this.highlightPkgOrImg("python") + "=" + python_version, this.highlightPkgOrImg("cuda-version") + "=" + cuda_version].join(" ");
                 var conda_channels = [rapids_channel, "conda-forge", "nvidia"]
-                    .map(ch => this.highlightFlag("-c") + " " + ch + " ")
+                    .map(ch => "-" + this.highlightFlag("c") + " " + ch + " ")
```

Comment:
I think the dashes are usually highlighted the same way as the flag name?
```suggestion
                    .map(ch => this.highlightFlag("-c") + " " + ch + " ")
```

My vim highlighting looks like:

![image](https://github.com/rapidsai/docs/assets/3943761/6eb281ae-2bc3-4edd-9fdb-90b60454e70c)

Reply:
interesting. okay. I will update in a bit. I just made them match the existing highlights.